components/GettingStarted.stories.jsx

```diff
@@ -0,0 +1,38 @@
+import { Linking, Text, View } from "react-native";
+
+const StarterComponent = () => (
+  <View style={{ padding: 16 }}>
+    <Text>
+      Thanks for trying out storybook, follow the{" "}
```

Comment:
@dannyhw when you have a moment, can you capitalize Storybook in here so that we don't accidentally introduce a minor typo? Thanks in advance.

Reply:
done!